src/components/disclosure/components/disclosure-activator/components/heading-wrapper/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactNode } from 'react'
+import { HeadingLevel } from './types'
+import s from './heading-wrapper.module.css'
+
+/**
+ * Given a heading level,
+ * Return a heading component at that level that will wrap children
+ * without adding styles.
+ */
+export function getHeadingWrapper(level: HeadingLevel) {
+	const Heading = level as React.ElementType
+	// eslint-disable-next-line react/display-name
+	return ({ children }: { children: ReactNode }) => (
+		<Heading className={s.headingMarginReset}>{children}</Heading>
```

Comment:
💭 [You could use the `Heading` component](https://github.com/hashicorp/dev-portal/blob/main/src/components/heading/types.ts#L19) and just pass a number for the level

Reply:
I think that'd add classes & styles that we might not want though 🤔 Could pass a `className` to reset them, similar to how `margin` is being reset here, but figured might be easier to not have to worry about the CSS differences.

I'll add a comment here with the rationale! Definitely happy to adjust if we feel strongly about using the existing `Heading` component for this use case, it certainly seems possible 👍

Reply:
Oh okay if the styles don't match up then disregard! 👍🏻